packages/next/client/page-loader.js

```diff
@@ -70,14 +69,32 @@ export default class PageLoader {
         return
       }
 
-      // Load the script if not asked to load yet.
       if (!this.loadingRoutes[route]) {
-        this.loadScript(route)
+        // Make sure we don't load a dependency that's already loaded
+        const loadedModules = new Set(
+          Array.from(document.getElementsByTagName('script'), el => {
```

Comment:
`Array.from` isn't supported in IE and will need polyfilled. Can we switch this out for something else?